trousers/github_comment.template

```diff
@@ -1,18 +1,21 @@
-PR build results:
+PRbuilds results:
 
 {% if "screenshots" in results %}
-screenshots
+**Screenshots**
 {{ " • ".join(links_for("screenshot")) }}
 {% endif %}
 
 {% if "exceptions" in results %}
-exceptions ({{ results["exceptions"]["raw_output"].count("EXCEPTION") }})
+**Exceptions** ({{ results["exceptions"]["raw_output"].count("EXCEPTION") }})
 {{ " • ".join(links_for("thrown-exception")) }}
 {% endif %}
 
 {% if "webpagetest" in results %}
-webpagetest ({{ results["webpagetest"]["raw_output"].count("Warning") }})
+**Webpagetest** ({{ results["webpagetest"]["raw_output"].count("Warning") }})
 {{ " • ".join(links_for("performanceComparisonSummary")) }}
 {% endif %}
 
--automated message
```

Comment:
Ahaa, sure! Will change for now.

In the long term it could be interested to run PRbuilds again, after every new commit in a PR. What do you think?